packs/rabbitmq/metadata.hcl

```diff
@@ -0,0 +1,11 @@
+app {
+  url    = "https://www.rabbitmq.com"
+  author = "@pondidum"
```

Comment:
We should clarify this, but this value is actually meant to be the author of the underlying app/contents of the pack. So in this case Pivotal Software for Rabbit I think?

(sorry, that's probably confusing)

Reply:
Ah, I read the docs a few times and wasn't certain. Will change once I'm at my laptop.

Reply:
Done